source/conclusions.rst

```diff
@@ -1,18 +1,18 @@
-Conclusions 
+Conclusions
 ***********
 
 Programming
 ===========
 
-The constructs we have learnt (loops, conditions, data structures) mean that we
-are far more expressive as programmers.
+The constructs we have learned (loops, conditions, data structures) mean that we
+can be very expressive as programmers.
 
-Combined with abstractions we can compose and recompose new programs.
+Combined with abstractions we can combine new programs together.
```

Comment:
It just felt quite functional-programming/computer-sciency to say "compose/recompose" rather than "combine"; happy to leave it if my intuition is squiffy on this one!